tyr-runner/src/main/resources/format-example.yaml

```diff
@@ -10,17 +10,21 @@ format:
     pattern: "\\[WFLY-\\d+\\]\\s+.*|WFLY-\\d+\\s+.*"
     message: "Wrong content of the title!"
   commit:
-    pattern: "\\[WFLY-\\d+\\]\\s+"
+    pattern: "\\[WFLY-\\d+\\]\\s+.*"
     message: "One of the commit messages has wrong format!"
   description:
     required rows:
       - pattern: "JIRA:\\s+https://issues.jboss.org/browse/WFLY-\\d+|https://issues.jboss.org/browse/WFLY-\\d+"
-        message: "The PR description must contain a link to the JIRA issue"
+        message: "The PR description must contain a link to the JIRA issue"    
+    optional rows:
+      - precondition: "\\[WFLY-\\s+.*\\]"
+        pattern: "\\[WFLY-\\d+\\]"
+        message: "JIRA issues can contain only numbers"
 
   commands:
     AddUserCommand: "ok\\s+to\\s+test$"
     RetestCommand: "retest\\s+this\\s+please$"
     RetestFailedCommand: "retest\\s+failed\\s+please$"
-
+    
   CI:
-    - TeamCity
+    - TeamCity    
```

Comment:
also won't block on this but beware of this kind of formatting changes in the future. They are often regarded as just unnecessary noise.